swap_meet/vendor.py

```diff
@@ -1,2 +1,105 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        # if no inventory, default to empty list
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        # always add item to inventory
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        # remove item if it exists
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
👍 We need to check the item is actually there before using remove, or else we would need to handle the error that could be thrown if we try to delete something that doesn't exist in the list. 

If we reorder the logic we can emphasize the main logic of this method by unindenting it and placing it in a more significant part of the function, rather than being just one branch of a condition.

```py
        if item in not self.inventory:
            return False

        self.inventory.remove(item)
        return item
```